content/release/ReleaseNotes346.md

```diff
@@ -0,0 +1,52 @@
+<!--
+title: "Contrast 3.4.6 - November 2017"
+description: "Contrast 3.4.6 November 2017"
+tags: "3.4.6 November Release Notes"
+-->
+
+Tame unruly applications with improved Assess Rules, unique display names and a new Protect rule for Node.js. 
+
```

Comment:
Should we be including anything about Node.js Protect yet since it's not officially out yet?